src/main/java/nextstep/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.qna.domain;
+
+import nextstep.users.domain.NsUser;
+
+public abstract class BaseEntity extends BaseTimeEntity {
+
+    private Long id;
+
+    private NsUser writer;
+
+    private boolean deleted = false;
+
+    protected BaseEntity(Long id, NsUser writer) {
+        this.id = id;
+        this.writer = writer;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public NsUser getWriter() {
+        return writer;
+    }
+
+    public void markAsDeleted() {
```

Comment:
```suggestion
    protected void markAsDeleted() {
```
👍 
단, 접근 제어자를 protected로 구현하는 것은 어떨까?